pkg/lightning/backend/local/engine_mgr.go

```diff
@@ -426,12 +433,6 @@ func (em *engineManager) resetEngine(ctx context.Context, engineUUID uuid.UUID)
 		if err = em.allocateTSIfNotExists(ctx, localEngine); err != nil {
 			return errors.Trace(err)
 		}
-		failpoint.Inject("mockAllocateTSErr", func() {
-			// mock generate timestamp error when reset engine.
-			localEngine.TS = 0
-			mockGRPCErr, _ := status.FromError(errors.Errorf("mock generate timestamp error"))
-			failpoint.Return(errors.Trace(mockGRPCErr.Err()))
-		})
```

Comment:
Why remove this?

Reply:
Oh I should let caller pass-in a TS, and skip this `allocateTSIfNotExists`

Reply:
https://github.com/pingcap/tidb/pull/52993/commits/8f94822663611acec74a5dad928deddc2f7533fa